docker/docker.mk

```diff
@@ -1,15 +1,7 @@
-#
-# Copyright 2021 The Dapr Authors
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#     http://www.apache.org/licenses/LICENSE-2.0
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
+# ------------------------------------------------------------
```

Comment:
This is going backwards. How did this happen?

Reply:
My bad. I resolved the merge conflict for this file with `git checkout --theirs` which did this. Will manually fix this up.